### ISSUE TITLE
Galacticraft Fuel Canister Exploit

### ISSUE THREAD

**Issue by evknucklehead** [sciguyryan/TheCrackPack#962]
Seems that under the right conditions, putting an Empty Fluid Canister into a Fluid Transposer with Fuel in it can cause the canister to overfill, resulting in one or more Partially Filled Fuel Canisters with 1001 fuel in them.

If one of these canisters is placed back into a Fluid Transposer set to empty mode, it will produce an infinite amount of Fuel very rapidly.

I have not tested if this is the case with the other fluids compatible with the GC Fluid Canisters. Server Operators may wish to manually blacklist the canisters from the Fluid Transposers unless updated versions of Thermal Expansion or Galacticraft fix this issue.

**Comment by evknucklehead** [comment]
Tested with Oil, and things get weirder. When filling the canisters, the empty canister stays in the item to fill slot while a properly filled oil canister comes out. This results in free canisters. Reversing the flow causes the infinite fluid problem, but also results in extra filled oil canisters, as the filled canister moves to the bottom slot with an empty canister taking its place in the top slot. Unlike the fuel canisters, however, they have the proper 1000 units of Oil in them. 

There aren't many other fluids that the canisters can hold, but I suspect that they also have similar problems.

I checked the Galacticraft changelogs, but there's no mention of a fluid duplication glitch being fixed. Haven't checked the Thermal Expansion changelogs, but updating that one appears to be a bit more involved than updating GC.
